crates/pop-contracts/src/templates.rs

```diff
@@ -75,80 +49,56 @@ impl ContractType {
 	PartialEq,
 	VariantArray,
 )]
-pub enum Template {
-	/// A minimal contract template.
+pub enum ContractTemplate {
+	/// A minimalist contract template.
 	#[default]
 	#[strum(
 		serialize = "standard",
 		message = "Standard",
 		detailed_message = "ink!'s 'Hello World': Flipper",
-		props(ContractType = "Examples")
+		props(TemplateType = "Examples")
```

Comment:
```suggestion
		props(Type = "Examples")
```
Would read much better as this, hence the suggestion to change the const value in pop_common::templates